Gopkg.toml

```diff
@@ -66,21 +72,21 @@ revision = "01aeca54ebda6e0fbfafd0a524d234159c05ec20"
 name = "github.com/docker/distribution"
 revision = "cd27f179f2c10c5d300e6d09025b538c475b0d51"
 [[override]]
-name = "github.com/docker/engine-api"
-revision = "dea108d3aa0c67d7162a3fd8aa65f38a430019fd"
+name = "github.com/docker/docker"
+version = "v1.13.1"
 [[override]]
 name = "github.com/docker/go-connections"
 revision = "f549a9393d05688dff0992ef3efd8bbe6c628aeb"
 [[override]]
 name = "github.com/docker/go-units"
 revision = "e30f1e79f3cd72542f2026ceec18d3bd67ab859c"
 [[override]]
+name = "github.com/docker/spdystream"
```

Comment:
are all of these other changes transitive dependenices? Or did all other deps get bumped up too?

Reply:
All transitive dependencies. I've explicitly listed them because dep doesn't parse godep and glide files in other repos. It won't automatically detect the versions the kubernetes requests.